rct229/ruletest_engine/ruletest_engine.py

```diff
@@ -508,6 +508,6 @@ def run_envelope_tests():
     Results of envelope stest are spit out to console
     """
 
-    envelope_test_json = "envelope_tests.json"
```

Comment:
Let's keep the `envelope_tests.json` to be the default in the developer branch. Could you change the `lighting_test_json` to `lighting_tests.json` as well? Thanks